src/Qwiq.Core.Soap/QueryFolder.cs

```diff
@@ -0,0 +1,24 @@
+using System.Linq;
+using Microsoft.Qwiq.Exceptions;
+
+namespace Microsoft.Qwiq.Client.Soap
+{
+    public class QueryFolder : Qwiq.QueryFolder
+    {
+        public QueryFolder(TeamFoundation.WorkItemTracking.Client.QueryFolder queryFolder)
```

Comment:
Should this not be `public`? The constructor exposes the underlying TFS type.

Reply:
Should be internal, will update.

Reply:
updated